src/cmakeTools.ts

```diff
@@ -208,12 +199,12 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
             const expandedConfigurePreset = await preset.expandConfigurePreset(this.folder.uri.fsPath,
                 configurePreset,
                 lightNormalizePath(this.folder.uri.fsPath || '.'),
-                this.sourceDir,
+                this.srcDir,
```

Comment:
If we have to rename this one, I'd like to stop abbreviating things.  `sourceDirectory` is better.

Reply:
I had to use sourceDir as it was in the API,  \src\api.ts
I assumed we don't change the API.
but I renamed the private one to _sourceDir to match the pattern